assets/js/modules/analytics/components/common/GA4Notice.js

```diff
@@ -27,18 +27,18 @@ import { __ } from '@wordpress/i18n';
 import SettingsNotice, { TYPE_INFO } from '../../../../components/SettingsNotice';
 import Link from '../../../../components/Link';
 
+const LearnMore = () => (
+	<Link
+		href="https://sitekit.withgoogle.com/documentation/ga4-analytics-property/"
+		external
+		inherit
+	>
+		{ __( 'Learn more here.', 'google-site-kit' ) }
+	</Link>
+);
```

Comment:
Let's move this into the `SettingsNotice` component and render it if a URL is passed via the `learnMore` prop:

```javascript
<SettingsNotice
    type={ ... }
    notice={ ... }
    learnMore="https://sitekit.withgoogle.com/documentation/ga4-analytics-property/"
/>
```

Reply:
@eugene-manuilov But the AC defines the API for this component! :smile: 

Doing what you say means the text would be fixed and the link would always be external. I'm not sure if this is acceptable (and there could be other things I'm missing)

Reply:
Ok, then maybe let's add it as an anonymous component to the LearnMore property directly?

Reply:
@danielgent similarly to how we do it for the Footer prop here: https://github.com/google/site-kit-wp/blob/12c099c53ee84a33d1486873ee815d478aedeea6/assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/index.js#L188-L199

Reply:
Done.  Honestly, I think `LearnMore` should just be `learnMore` and be a node but it's like this on the AC so I've done it like this :smile: